tensorrt_llm/serve/openai_protocol.py

```diff
@@ -633,6 +640,17 @@ def check_suffix(cls, data):
             raise ValueError("suffix is not supported")
         return data
 
+    @model_validator(mode="before")
+    @classmethod
+    def check_cache_salt_support(cls, data):
```

Comment:
It is better to use a field_validator here, as this validator only cares about a single field `cache_salt`; model_validator is a validator for accessing multiple fields.